mp/src/vgui2/vgui_controls/AnimationController.cpp

```diff
@@ -1023,10 +1063,25 @@ bool AnimationController::StopAnimationSequence( Panel *pWithinParent, const cha
 	return true;
 }
 
+bool AnimationController::StopAnimationSequenceForPanel(Panel *pPanel, const char *sequenceName)
+{
+    Assert(pPanel);
+
+    // lookup the symbol for the name
+    UtlSymId_t seqName = g_ScriptSymbols.Find(sequenceName);
+    if (seqName == UTL_INVAL_SYMBOL)
+        return false;
+
+    // remove the existing command from the queue
+    RemoveQueuedAnimationCommandsForPanel(seqName, pPanel);
+
+    return true;
+}
+
 //-----------------------------------------------------------------------------
 // Purpose: Runs a custom command from code, not from a script file
 //-----------------------------------------------------------------------------
-void AnimationController::CancelAnimationsForPanel( Panel *pWithinParent )
+void AnimationController::CancelAnimations( Panel *pWithinParent )
```

Comment:
You now need to verify things are calling the right function. The old cancel method was checking the parent and finding the child, but since you renamed this to something else, was there something else in the code calling the old function name? The new function you added took its old name and does slightly different behavior; this could be a regression if something was using the old method.

Reply:
I went through the code first and ensured nothing was calling this function before making this change.